cmd/mungedocs/mungedocs.go

```diff
@@ -46,10 +46,10 @@ Examples:
 	// TODO: allow selection from command line. (e.g., just check links in the examples directory.)
 	allMunges = []munge{
 		{"table-of-contents", updateTOC},
+		{"unversioned-warning", updateUnversionedWarning},
```

Comment:
checkHeaderLines needs to run after this.
It adds blank lines above headings, but then this adds unversionedWarnings which messes up with those blank lines.